### ISSUE TITLE
Fix for Nashorn deprecation warning

### ISSUE THREAD

**Issue by jimkyndemeyer** [jimkyndemeyer/js-graphql-intellij-plugin#287]
The plugin uses Nashorn to execute minimatch JavaScript code to be compatible with `graphql-config`. But Nashorn has been deprecated in JDK 11:

```
Warning: Nashorn engine is planned to be removed from a future JDK release
```

Related issue: https://github.com/GatorEducator/gatorgradle/issues/39

**Comment by denofevil** [comment]
You can try using https://github.com/angelozerr/minimatch.java instead

**Comment by jimkyndemeyer** [comment]
Fixed in the upcoming 2.3.0 release.
